REVIEWS.md

```diff
@@ -67,8 +75,14 @@ Gitter is not owned by github, AFAIK. They just integrate. (Line 203).
 Why prioritize it over others? For example, Slack is another service
 that provides similar functionality. (We use gitter, but still.)
 
+> R/ We agree with the reviewer. In the present version we refined the paragraph making clear 
+that Gitter is not only chat system integrated with Github. We also included Gitter as an example of 
+chat system but leaving the space open for other systems.   
```

Comment:
maybe rephrase: Gitter serves, amongst several alternatives, as a popular example for a github integrated chat system.